### DISCUSSION TITLE
help: docker release pipeline does not update the latest tag

### DISCUSSION THREAD

**Discussion by romange** [dragonflydb/dragonfly#122] [Q&A] [answered]
I do not find the root cause. 
My docker release pipeline seems to work well, https://github.com/dragonflydb/dragonfly/blob/main/.github/workflows/docker-release.yml
but for some reason, it does not update the "latest" tag. I had to manually tag and push it. 
The pipeline logs do not show anything suspicious as well. 

Ideas? 🤔

**Comment by tamcore** [comment, marked as answer]
How exactly did you come to the conclusion that the latest tag in the registry is wrong? My ad-hoc guess would be, that you tried to run a container with the latest-tag and your local Docker instance used the already present image 🙂 `docker run --pull always` would ensure the image is always pulled (or at least verified against the local cache).

**Reply by romange** [reply]
I thought I did an explicit `pull` but now I doubt myself. I will rerun the docker pipeline and will double check.

**Reply by romange** [reply]
For the reference, current latest has ` "LastTagTime": "2022-06-08T11:24:02.182806883+03:00"`

**Reply by romange** [reply]
@TamCore , ok so a different question. I've run a docker pipeline manually. I print there `${{ github.event.inputs.PRERELEASE }}` and `${{ github.event.release.prerelease }}` in `Extract artifacts` step. 
seems like I would expect that these fields will be false or non-present. However, for some reason, the pipeline executes `Build pre-release image` step and omits `Build release image` step. Do you know what can be wrong?

**Reply by romange** [reply]
you can see the log here: https://github.com/dragonflydb/dragonfly/runs/6792175208?check_suite_focus=true

**Reply by romange** [reply]
nm, I think I know why this happens
